### PR TITLE
Add descriptions to the graphs and add a place holder for mancha urbana

### PR DESCRIPTION
**Pull request by louloubadillo**

### CODE REVIEW

File: formaurbana.html [hunk 1]
```diff
@@ -51,8 +62,7 @@
     </div>
 
 
-    <h1>Aquí va la parte de mancha urbana</h1>
-    <!-- si la tuvieramos xdxd -->
+    <iframe src="https://ctrl-alt-tec.hackclub.com" frameborder="0"></iframe>
```
Comment (edvilme) [inline formaurbana.html:65]: Si, aquí se ve el hackeo de Lalo

**Review by edvilme — APPROVED** [review]
# Reviewed, I approve
image

File: formaurbana.html [hunk 2]
```diff
@@ -22,6 +22,17 @@
...
     <!--Charts-->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js"charset="utf-8"></script>
+    <style>
+        body{
+            height: 100vh;
+            display: flex;
+            flex-direction: column;
+        }
+        iframe{
+            flex: 1
```
Comment (franspaco) [inline formaurbana.html:32]: ;

**Review by franspaco — COMMENTED** [review]
🤔

File: formaurbana.html (hunk 2 above)
Comment (edvilme) [reply formaurbana.html:32]: weird `flex` but ok, I guess :P

**Pull request merged**